INSTALL.md

```diff
@@ -41,8 +41,8 @@ All contributed Python code, must use the [black](https://github.com/ambv/black)
 git clone --recursive git@github.com:Materials-Consortia/optimade-python-tools.git
 cd optimade-python-tools
 
-# Ensure a Python>=3.7 (virtual) environment (example below using Anaconda/Miniconda)
-conda create -n optimade python=3.7
+# Ensure a Python>=3.8 (virtual) environment (example below using Anaconda/Miniconda)
+conda create -n optimade python=3.8
```

Comment:
```suggestion
conda create -n optimade python=3.10
```

Reply:
My reason for sticking with an older python version is that not all python packages seem to have [support for 3.10 yet.](https://pyreadiness.org/3.10/l) So I did not want to give people setting up their environment for the OPTIMADE python tools problems when they later want to use a package which does not support python 3.10 yet. 
I do not have so much experience with developing python software, so I do not know whether this would be a big problem in practice.

Reply:
We are instructing people to make a fresh conda environment anyway so I don't think we need to worry about that. All of our dependencies at least support py310